source/mapping-rules/mapping-rule-options/mapping-rule-options.txt

```diff
@@ -57,6 +57,12 @@ mapping rule options and the mapping rules they apply to.
    * - Merge fields into the parent
      - Merges the table rows as fields at the parent collection level.
      - :guilabel:`Embedded documents` 
+   * - Mapping rule filter
+     - Explicitly include rows where an expression returns true.
+     - :guilabel:`Add mapping rule filter`
```

Comment:
The "applies to" column is used to specify which type of mapping rule this option can be used on. For array filters it is all three.